docs/integrations/unity-catalog-daft.md

```diff
@@ -1,3 +1,119 @@
 # Unity Catalog Daft Integration
 
-TODO
+This page shows you how to use Unity Catalog with Daft.
```

Comment:
I think it makes more sense to name this file just `daft.md`. everything is in unitycatalog and so it does make sense to have the prefix `unity-catalog-` in an individual page with the url being `docs.unitycatalog.io/integrations/unity-catalog-daft/`. Just `docs.unitycatalog.io/integrations/daft/` makes more sense.